src/test/java/org/opensearch/flowframework/rest/FlowFrameworkRestApiIT.java

```diff
@@ -48,15 +46,12 @@
 import static org.opensearch.flowframework.common.CommonValue.WORKFLOW_ID;
 
 public class FlowFrameworkRestApiIT extends FlowFrameworkRestTestCase {
-    private static AtomicBoolean waitToStart = new AtomicBoolean(true);
 
     @Before
     public void waitToStart() throws Exception {
         // ML Commons cron job runs every 10 seconds and takes 20+ seconds to initialize .plugins-ml-config index
-        // Delay on the first attempt for 25 seconds to allow this initialization and prevent flaky tests
-        if (waitToStart.getAndSet(false)) {
-            CountDownLatch latch = new CountDownLatch(1);
-            latch.await(25, TimeUnit.SECONDS);
+        if (!indexExistsWithAdminClient(".plugins-ml-config")) {
```

Comment:
Might be nice to use the ML Commons `CommonValue.ML_CONFIG_INDEX` constant here (static import).